src/olympia/devhub/tests/test_tasks.py

```diff
@@ -579,9 +579,9 @@ def test_amo_validator_addons_linter_error(self):
         self.create_switch('addons-linter')
 
         # This test assumes that `amo-validator` doesn't correctly
-        # validate a missing id in manifest.json
+        # validate a invalid id in manifest.json
         result = tasks.validate_file_path(
-            get_addon_file('invalid_webextension_missing_id.xpi'),
+            get_addon_file('invalid_webextension_invalid_id.xpi'),
```

Comment:
We validate IDs but don't require them. Should we at least file a bug to move this into the linter? (Or maybe it already does so)